### ISSUE TITLE
Translated title of main buttons partly hidden

### ISSUE THREAD

**Issue by Cwpute** [itsdrnoob/DataMonitor#209]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/itsdrnoob/DataMonitor/releases/latest).
- [X] I made sure that there are no existing issues - [open](https://github.com/itsdrnoob/DataMonitor/issues) or [closed](https://github.com/itsdrnoob/DataMonitor/issues?q=is%3Aissue+is%3Aclosed).
- [X] I have taken the time to fill in all the required details in an appropriate descriptive way. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.

### Affected app version

2.3.2

### Steps to reproduce the bug

1. Select a translated versions of DataMonitor (e.g. French)
2. Look at the text in the main four buttons in the bottom menu:
![edited screenshot](https://github.com/itsdrnoob/DataMonitor/assets/80604512/7ae8b19c-2d75-4501-9ea1-27fc04a213e8)

### Expected behavior

The title of those buttons should always be seen in their entirety as they are the main options the user is presented with.
If the app allowed the text here to wrap to a second line, it should help with most translated cases.

### Actual behavior

Title is partly hidden

### Affected Android/Custom ROM version

Android 11

### Affected device model

Nokia TA-1157

### Additional context

My screen resolution is 720x1520 pixels.

**Comment by Cwpute** [comment]
It could be argued that the translations account for the space available, but translators on Weblate don't have much context to work with.
Also there are some languages i bet won't be able to find short enough words that fit in a single line.

Another alternative would be dropping the names of these sections entirely, to only have explicit enough icons, which would solve the problem altogether, save some work for translators and give the app a cleuner look.

**Comment by itsdrnoob** [comment]
I agree that some translations would not fit in a single line, but the title is also shown on the top (toolbar), which would be enough to provide clarity.

**Comment by Cwpute** [comment]
> I agree that some translations would not fit in a single line, but the title is also shown on the top (toolbar), which would be enough to provide clarity.

Indeed. So the titles below the icons aren't needed anymore then ?

**Comment by itsdrnoob** [comment]
> Indeed. So the titles below the icons aren't needed anymore then ?

Not exactly, some users may rely on the titles below the icons for context, especially if they're not familiar with the icons. It would provide clarity without relying solely on the toolbar for title visibility. While I do acknowledge that in certain translations, the text is not entirely visible, removing it completely would not be an ideal solution.

**Comment by ihatemakinganaccount** [comment]
I don't know if this has been resolved yet, but if it wasn't, why not just make space for two rows of text if it's needed?
